### DISCUSSION TITLE
Error with duplicate app creation and inaccessible/undeletable app due to bug

### DISCUSSION THREAD

**Discussion by luigiPinnaAevo** [Budibase/budibase#9326] [Help]
Hello,

I am having an issue with the creation app tool where a bug has caused the creation of one app with the same name as one already present. One of these apps is not accessible or deletable because it has a bug and gives an error  (img below).

![budiBugApp](https://user-images.githubusercontent.com/99037506/212059185-d72f1f57-6c36-4aec-96a3-4433eed99fb5.png)
 
I want to be able to delete the app created by the bug.

Can anyone please help me resolve this issue? 

Thank you.

**Comment by luigiPinnaAevo** [comment]
Any suggestions?

**Reply by melohagan** [reply]
Hey @luigiPinnaAevo 

Are you a cloud user? If so, what is your tenant id?

**Reply by luigiPinnaAevo** [reply]
Hi @melohagan,

yes, I'm a cloud user. But I cannot share the tenant id because it's in our customer account. 
Are there other ways to solve this issue?

Thanks

**Reply by melohagan** [reply]
Can you email the tenant id through to support@budibase.com ?

Just for the sake of clarity, the tenant id is the part of your app URL, e.g. `bb.budibase.app` has the tenant id 'bb'
